test/include/dlaf_test/comm_grids/grids_6_ranks.h

```diff
@@ -73,11 +73,6 @@ class CommunicatorGrid6RanksCAPIEnvironment : public CommunicatorGrid6RanksEnvir
     if (comm_grids.empty()) {
       comm::Communicator world(MPI_COMM_WORLD);
 
-      // Leave comm_grids empty if invoked with only one rank.
-      // Useful to debug local algorithms that otherwise are executed independently on multiple ranks.
-      if (world.size() == 1)
-        return;
-
```

Comment:
Was this causing problems in the tests?

Reply:
No. But it's a leftover from copy-pasting from `CommunicatorGrid6RanksEnvironment`. Maybe we should replace it with an assertion that we have 6 ranks?

Reply:
The assertion is already done when constructing the grids.